python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -1,10 +1,9 @@
 from django.shortcuts import render_to_response
+from django.template import RequestContext
+from django.conf import settings
 
 def home(request):
-    return render_to_response('home.html')
+    return render_to_response('home.html', dict(debug=settings.DEBUG))
```

Comment:
Ohh, now looking at that, I think it was because we didn't have `INTERNAL_IPS` set correctly at the time.